pyproject.toml

```diff
@@ -50,10 +50,9 @@ tidal = "mopidy_tidal:Extension"
 
 [tool.pytest.ini_options]
 markers = [
-    "gt_3_7: Mark a test as requiring python > 3.7.",
-    "gt_3_8: Mark a test as requiring python > 3.8.",
-    "gt_3_9: Mark a test as requiring python > 3.9.",
```

Comment:
Are we not supporting 3.9?

Reply:
It's more that the marker isn't useful as 3.9 is the lowest python we support (so this would just mark everything).

Incidentally mopidy's develop branch now requires python 3.11, although the last release still claims to support 3.7.